src/ui/core/view.ios.ts

```diff
@@ -69,37 +54,38 @@ function handleUIAccessibilityElementFocusedNotification(view: UIView, tnsView:
     return;
   }
 
-  const selfView = new WeakRef<UIView>(view);
   const selfTnsView = new WeakRef<View>(tnsView);
 
-  const observer = nsApp.ios.addNotificationObserver(UIAccessibilityElementFocusedNotification, (args) => {
+  let observer = nsApp.ios.addNotificationObserver(UIAccessibilityElementFocusedNotification, (args: NSNotification) => {
```

Comment:
Question: Do we `addNotificationObserver` for each view with `accessible=true`? I'd think a single focus observer would do.

Reply:
We use the `args`-object below to see which views received or lost the focus.
We'd just be moving the callback to a TNS `Observable`.